apps/citizen-func/utils/redis_storage.ts

```diff
@@ -1,205 +1,98 @@
-import * as E from "fp-ts/lib/Either";
-import { pipe } from "fp-ts/lib/function";
+import { constTrue, pipe } from "fp-ts/lib/function";
 import * as O from "fp-ts/lib/Option";
 import * as TE from "fp-ts/lib/TaskEither";
-import { RedisClient } from "redis";
+import * as redis from "redis";
 
-/**
- * Parse a Redis given string reply.
- *
- * @see https://redis.io/topics/protocol#simple-string-reply.
- */
-const givenStringReply = (
-  err: Error | null,
-  reply: string | undefined,
-  message: string
-): E.Either<Error, boolean> =>
-  pipe(
-    err,
-    O.fromNullable,
-    O.map(E.left),
-    O.getOrElse(() => E.right(reply === message))
-  );
-
-/**
- * Parse a Redis single string reply.
- *
- * @see https://redis.io/topics/protocol#simple-string-reply.
- */
-const singleStringReply = (
-  err: Error | null,
-  reply: "OK" | undefined
-): E.Either<Error, boolean> =>
-  pipe(
-    err,
-    O.fromNullable,
-    O.map(E.left),
-    O.getOrElse(() => E.right(reply === "OK"))
-  );
-
-/**
- * Parse a Redis single string reply.
- *
- * @see https://redis.io/topics/protocol#simple-string-reply.
- */
-const singleValueReply = (
-  err: Error | null,
-  reply: string | null
-): E.Either<Error, O.Option<string>> =>
-  pipe(
-    err,
-    O.fromNullable,
-    O.map(E.left),
-    O.getOrElse(() => E.right(O.fromNullable(reply)))
-  );
-
-/**
- * Parse a Redis integer reply.
- *
- * @see https://redis.io/topics/protocol#integer-reply
- */
-const integerReply = (
-  err: Error | null,
-  reply: unknown,
-  expectedReply?: number
-): E.Either<Error, boolean> => {
-  if (err) {
-    return E.left<Error, boolean>(err);
-  }
-  if (expectedReply !== undefined && expectedReply !== reply) {
-    return E.right<Error, boolean>(false);
-  }
-  return E.right<Error, boolean>(typeof reply === "number");
-};
-
-const isTrue = (b: boolean): b is true => b === true;
+const isSetResponseOk = (response: string): boolean => response === "OK";
+const isDelResponseOk = (response: number): boolean => response >= 0;
```

Comment:
We always used utils like these https://github.com/pagopa/io-functions-cgn/blob/4df37c956d23a4aa2364a3017e4398bc71a81f18/utils/redis_storage.ts#L25 to do this kind of check. Async or sync. I'd stick to that convention.

Reply:
[491deef](https://github.com/pagopa/io-messages/pull/12/commits/491deefe11b65f378a886272732bf3b3b00bc8de)

Reply:
@michaeldisaro I removed the del utility cause we never use [bac1a14](https://github.com/pagopa/io-messages/pull/12/commits/bac1a14da123aae8ad1822e0d07abeed805cb03c)